Sources/SwiftInspectorCommands/InitializerCommand.swift

```diff
@@ -71,9 +71,9 @@ final class InitializerCommand: ParsableCommand {
 
   private func outputString(from statement: InitializerStatement) -> String {
     if typeOnly {
-      return statement.parameters.map { $0.typeName }.joined(separator: " ")
+      return statement.parameters.map { $0.typeNames.joined(separator: " ") }.joined(separator: " ")
```

Comment:
Also while we are thinking about edge cases like this, what happens if the type is a tuple? Or a closure?

Reply:
Good call! Let me add more tests and try those use cases.

Reply:
For a tuple we're handling this correctly. For a closure we're just returning the types on the closure itself but it's somewhat weird of a syntax since it'll just list the types.

I created this https://github.com/fdiaz/SwiftInspector/issues/23 to track this since I think this might be a bigger change.

Reply:
typle commit https://github.com/fdiaz/SwiftInspector/pull/22/commits/a12e7637a8120e5c9ee2ac0a6761767ae0ebca64